ci/run_envoy_docker.sh

```diff
@@ -86,6 +86,7 @@ docker run --rm \
        -e SYSTEM_PULLREQUEST_TARGETBRANCH \
        -e SYSTEM_PULLREQUEST_PULLREQUESTNUMBER \
        -e GCS_ARTIFACT_BUCKET \
+       -e GH_ACCESS_TOKEN \
```

Comment:
name this `GITHUB_TOKEN`? That is already a widely used name.

Reply:
Makes sense. I've also renamed other uses of GH_ACCESS_TOKEN to GITHUB_TOKEN in the repo.